src/plugins/formActions/EditPropertiesAction.js

```diff
@@ -76,6 +76,13 @@ export default class EditPropertiesAction extends PropertiesAction {
         }
     }
 
+    /**
+     * @private
+     */
+    _onCancel() {
+        //noop
```

Comment:
ahh thanks, 
there are more like this tracked here, https://github.com/nasa/openmct/issues/4864